src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http2.cs

```diff
@@ -139,7 +139,8 @@ public async Task Http2_StreamResetByServerBeforePrefix_RequestFailsWithGoawayPr
 
                 Http2LoopbackConnection connection = await server.AcceptConnectionAsync(timeout: null);
                 _ = await connection.ReadSettingsAsync();
-
+                // Wait until client starts sending request
+                Frame frame = await connection.ReadFrameAsync(TestHelper.PassingTestTimeout);
```

Comment:
Nit: we don't use the frame returned.

```suggestion
                _ = await connection.ReadFrameAsync(TestHelper.PassingTestTimeout);
```

Reply:
updated. I originally played with the frame type but I did not use it at the end.